addons/web/static/tests/helpers/test_utils.js

```diff
@@ -97,6 +97,18 @@ odoo.define('web.test_utils', async function (require) {
         return testUtilsDom.returnAfterNextAnimationFrame();
     }
 
+    /**
+     * Returns a promise that will be resolved after the tick after the nextAnimationFrame.
+     * This has to be used instead of nextTick if it is sure that the call will be useless when the
+     * situtation will be full owl: at some point all calls to owlCompatibilityExtraNextTick will
+     * be removed.
```

Comment:
The doc is not wrong, but it doesn't perfectly fit to the function's purpose either. The function itself should directly call nextTick (instead of replicating what it does). The purpose of this function is that, during the temporary period where we mix both legacy and owl implementations, we sometimes need more that a single nextTick (e.g. you click on a button, but the DOM is updated after two nextTick). The number of required nextTick depends on the number of Owl-legacy-Owl or legacy-Owl-legacy (can't remember which one it is) we have. We could use nextTick() directly in the tests, but it means that they would stay in the code forever. Using another function will allow in the future, when everything will be converted, to simply remove all lines calling owlCompatibilityExtraNextTick.

So the fact that it actually waits for the nextAnimationFrame is actually an implementation detail of nextTick itself. It should not be mentionned in this function's docstring, imho.

Btw, a similar explanation could be added to the commit message.